src/teehr/visualization/dataframe_accessor.py

```diff
@@ -28,15 +32,60 @@ class TEEHRDataFrameAccessor:
     def __init__(self, pandas_obj):
         """Initialize the class."""
         self._df = pandas_obj
+        self._df.attrs = pandas_obj.attrs
         self._validate(pandas_obj)
 
     @staticmethod
     def _validate(obj):
-        """Validate the DataFrame object columns."""
-        if "location_id" not in obj.columns:
-            raise AttributeError("Must have 'location_id'.")
-        if obj.index.size == 0:
-            raise AttributeError("DataFrame must have data.")
+        """Validate the DataFrame object."""
```

Comment:
I think we could/should use the Pandera validation methods we use in the loading code.  It would make sense to have a "schema" defined for each table that we can use more broadly.  Lets discuss.

Reply:
Per our discussion on 10/23 -- I think we benched the Pandera validation approach in favor of just validating the fields (for now). Let me know if anything has changed here.